<?xml version='1.0' encoding='UTF-8'?>
<ui version="4.0">
 <class>Transfer</class>
 <widget class="QDialog" name="Transfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutRoot"><property name="leftMargin"><number>40</number></property><property name="topMargin"><number>30</number></property><property name="rightMargin"><number>40</number></property><property name="bottomMargin"><number>30</number></property><item><spacer name="verticalSpacerTop"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint"><size><width>20</width><height>40</height></size></property></spacer></item><item><layout class="QHBoxLayout" name="horizontalLayoutCenter"><item><spacer name="horizontalSpacerLeft"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint"><size><width>40</width><height>20</height></size></property></spacer></item><item><widget class="QFrame" name="loginCard"><property name="minimumSize"><size><width>1076</width><height>750</height></size></property><layout class="QVBoxLayout" name="cardLayout"><property name="spacing"><number>0</number></property><property name="leftMargin"><number>0</number></property><property name="topMargin"><number>0</number></property><property name="rightMargin"><number>0</number></property><property name="bottomMargin"><number>0</number></property><item><widget class="QWidget" name="contentWidget"><property name="minimumSize"><size><width>1076</width><height>750</height></size></property><property name="maximumSize"><size><width>1076</width><height>750</height></size></property><widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblErrorDisplay">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>650</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red; font-size: 14px;</string>
   </property>
   <property name="text">
    <string />
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>681</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QLineEdit" name="leAmount">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="placeholderText">
       <string>0.00</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblAmountLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Amount:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leTargetAccount">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>18</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblTargetLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Receiver Account Number:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelTransfer">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string />
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>349</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Funds Transfer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget></item></layout></widget></item><item><spacer name="horizontalSpacerRight"><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="sizeHint"><size><width>40</width><height>20</height></size></property></spacer></item></layout></item><item><spacer name="verticalSpacerBottom"><property name="orientation"><enum>Qt::Vertical</enum></property><property name="sizeHint"><size><width>20</width><height>40</height></size></property></spacer></item></layout></widget>
 <resources />
 <connections />
</ui>